demos/tests/run_tests.py

```diff
@@ -149,11 +149,17 @@ def main():
         dl_dir = prepare_models(auto_tools_dir, args.downloader_cache_dir, args.mo, global_temp_dir, demos_to_test)
 
         num_failures = 0
+        
+        pythonpath_args = (os.environ['PYTHONPATH'], args.demo_build_dir)
+        
+        if os.name == "nt":
+            PYTHONPATH = {'PYTHONPATH': "{};{};".format(*pythonpath_args)}
+        else:
+            PYTHONPATH = {'PYTHONPATH': "{}:{}/lib".format(*pythonpath_args)}
```

Comment:
> `if os.name == "nt":`

Consider using `os.pathsep` for that instead.

Reply:
Done.